### ISSUE TITLE
CUDA not supported on NVIDIA GeForce RTX 3080

### ISSUE THREAD

**Issue by adrenaline21** [taichi-dev/taichi#1980]
```sh
~$ ti example stable_fluid
[Taichi] mode=release
[Taichi] preparing sandbox at /tmp/taichi-y2j8vvc1
[Taichi] version 0.7.0, llvm 10.0.0, commit 0298bca3, linux, python 3.6.9

*******************************************
**      Taichi Programming Language      **
*******************************************

Docs:   https://taichi.rtfd.io/en/stable
GitHub: https://github.com/taichi-dev/taichi
Forum:  https://forum.taichi.graphics

Running example stable_fluid ...
[Taichi] Starting on arch=cuda
[Taichi] materializing...
'sm_86' is not a recognized processor for this target (ignoring processor)
'sm_86' is not a recognized processor for this target (ignoring processor)
'sm_86' is not a recognized processor for this target (ignoring processor)
'sm_86' is not a recognized processor for this target (ignoring processor)
'sm_86' is not a recognized processor for this target (ignoring processor)
'sm_86' is not a recognized processor for this target (ignoring processor)
[E 10/21/20 14:12:04.059] [cuda_driver.h:operator()@80] CUDA Error CUDA_ERROR_INVALID_PTX: a PTX JIT compilation failed while calling module_load_data_ex (cuModuleLoadDataEx)


***********************************
* Taichi Compiler Stack Traceback *
***********************************
/home/darkartist/.local/lib/python3.6/site-packages/taichi/core/../lib/taichi_core.so: taichi::Logger::error(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, bool)
/home/darkartist/.local/lib/python3.6/site-packages/taichi/core/../lib/taichi_core.so: taichi::lang::CUDADriverFunction<void**, char const*, unsigned int, unsigned int*, void**>::operator()(void**, char const*, unsigned int, unsigned int*, void**)
/home/darkartist/.local/lib/python3.6/site-packages/taichi/core/../lib/taichi_core.so: taichi::lang::JITSessionCUDA::add_module(std::unique_ptr<llvm::Module, std::default_delete<llvm::Module> >)
/home/darkartist/.local/lib/python3.6/site-packages/taichi/core/../lib/taichi_core.so: taichi::lang::TaichiLLVMContext::set_struct_module(std::unique_ptr<llvm::Module, std::default_delete<llvm::Module> > const&)
/home/darkartist/.local/lib/python3.6/site-packages/taichi/core/../lib/taichi_core.so: taichi::lang::StructCompilerLLVM::run(taichi::lang::SNode&, bool)
/home/darkartist/.local/lib/python3.6/site-packages/taichi/core/../lib/taichi_core.so: taichi::lang::Program::materialize_layout()
/home/darkartist/.local/lib/python3.6/site-packages/taichi/core/../lib/taichi_core.so: taichi::lang::layout(std::function<void ()> const&)
/home/darkartist/.local/lib/python3.6/site-packages/taichi/core/../lib/taichi_core.so(+0x541229) [0x7fbeee718229]
/home/darkartist/.local/lib/python3.6/site-packages/taichi/core/../lib/taichi_core.so(+0x3d5def) [0x7fbeee5acdef]
/usr/bin/python3: ) [0x50a4a5]
/usr/bin/python3: _PyEval_EvalFrameDefault
/usr/bin/python3: ) [0x507be4]
/usr/bin/python3: ) [0x509900]
/usr/bin/python3: ) [0x50a2fd]
/usr/bin/python3: _PyEval_EvalFrameDefault
/usr/bin/python3: ) [0x507be4]
/usr/bin/python3: ) [0x509900]
/usr/bin/python3: ) [0x50a2fd]
/usr/bin/python3: _PyEval_EvalFrameDefault
/usr/bin/python3: ) [0x507be4]
/usr/bin/python3: _PyFunction_FastCallDict
/usr/bin/python3: ) [0x594a01]
/usr/bin/python3: ) [0x54a971]
/usr/bin/python3: PyObject_Call
/usr/bin/python3: _PyEval_EvalFrameDefault
/usr/bin/python3: ) [0x507be4]
/usr/bin/python3: ) [0x509900]
/usr/bin/python3: ) [0x50a2fd]
/usr/bin/python3: _PyEval_EvalFrameDefault
/usr/bin/python3: ) [0x5095c8]
/usr/bin/python3: ) [0x50a2fd]
/usr/bin/python3: _PyEval_EvalFrameDefault
/usr/bin/python3: ) [0x507be4]
/usr/bin/python3: ) [0x516069]
/usr/bin/python3: ) [0x50a12f]
/usr/bin/python3: _PyEval_EvalFrameDefault
/usr/bin/python3: ) [0x507be4]
/usr/bin/python3: ) [0x509900]
/usr/bin/python3: ) [0x50a2fd]
/usr/bin/python3: _PyEval_EvalFrameDefault
/usr/bin/python3: ) [0x507be4]
/usr/bin/python3: ) [0x509900]
/usr/bin/python3: ) [0x50a2fd]
/usr/bin/python3: _PyEval_EvalFrameDefault
/usr/bin/python3: ) [0x507be4]
/usr/bin/python3: ) [0x509900]
/usr/bin/python3: ) [0x50a2fd]
/usr/bin/python3: _PyEval_EvalFrameDefault
/usr/bin/python3: ) [0x507be4]
/usr/bin/python3: ) [0x509900]
/usr/bin/python3: ) [0x50a2fd]
/usr/bin/python3: _PyEval_EvalFrameDefault
/usr/bin/python3: ) [0x507be4]
/usr/bin/python3: ) [0x588d41]
/usr/bin/python3: PyObject_Call
/usr/bin/python3: _PyEval_EvalFrameDefault
/usr/bin/python3: ) [0x507be4]
/usr/bin/python3: _PyFunction_FastCallDict
/usr/bin/python3: ) [0x594a01]
/usr/bin/python3: ) [0x54a971]
/usr/bin/python3: _PyObject_FastCallKeywords
/usr/bin/python3: ) [0x50a433]
/usr/bin/python3: _PyEval_EvalFrameDefault
/usr/bin/python3: ) [0x5095c8]
/usr/bin/python3: ) [0x50a2fd]
/usr/bin/python3: _PyEval_EvalFrameDefault
/usr/bin/python3: ) [0x507be4]
/usr/bin/python3: PyEval_EvalCode
/usr/bin/python3: ) [0x634e72]
/usr/bin/python3: PyRun_FileExFlags
/usr/bin/python3: PyRun_SimpleFileExFlags
/usr/bin/python3: Py_Main
/usr/bin/python3: main
/lib/x86_64-linux-gnu/libc.so.6: __libc_start_main
/usr/bin/python3: _start
```

**Comment by yuanming-hu** [comment]
Thanks for reporting. This is fixed in #1983. @adrenaline21 Please try out v0.7.1 and let me know if it works.

**Comment by adrenaline21** [comment]
Works fine now.
